### PR TITLE
2.12: new Scala SHA

### PR DESCRIPTION
**Pull request by SethTisue**
~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6145/~
~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6146/~
~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6163/~
~https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6167/~
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6169/ - that fixed silencer, next let's try acyclic
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6171/

### CODE REVIEW

**Comment by SethTisue** [comment]
@dwijnand I've assigned this to you because I've already seen in local testing that some repos fail, which you can see for yourself with `./narrow silencer acyclic`, and because it looks like a consequence of the configurable-warnings and I'm not volunteering to work the necessary changes through. probably the necessary changes already had to be done once on 2.13.x and are out there somewhere to be cherry-picked

**Comment by dwijnand** [comment]
Yeah https://github.com/ghik/silencer/pull/33

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6165/console

```
FAILURES (UNEXPECTED): silencer,acyclic,zinc,linter,splain,scala-rewrites,scala-refactoring
BLOCKING DOWNSTREAM: silencer (26), acyclic (15), linter (1), zinc (1)
```

fwiw, silencer is blocking less of the build than I was thinking when we were discussing this earlier today

**Comment by dwijnand** [comment]
My 0 LOC fix passed a local test of compiling with the mergely you're using here, so I pushed it and switched the config to use it and kicked off a Jenkins run of it (6167).

**Comment by dwijnand** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6​17​1/

File: proj/silencer.conf [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // https://github.com/ghik/silencer.git#787144f5557421017263c1ea36c22f4c078d24cb  # was master
```
Comment (SethTisue) [inline proj/silencer.conf]: you'll need to edit this line as well, in order for `advance` not to wipe out the change you made below. usually I edit this line first, then run `./advance silencer` to automatically update the line below

**Comment by dwijnand** [comment]
Even better:

```
SUCCEEDED: 200
FAILURES (UNEXPECTED): scalameta,zinc,linter,splain,scala-rewrites,scala-refactoring
BLOCKING DOWNSTREAM: scalameta (5), linter (1), zinc (1)
FAILED: 7
BLOCKED, DID NOT RUN: 7
TOTAL: 214
```

**Comment by SethTisue** [comment]
If that's good enough in your estimation, you could add the repos as expected-to-fail (in `report/Report.scala`), to make the PR mergeable, then open an issue to follow up post-2.12.13 (and assign it to me).

**Comment by dwijnand** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-jdk8-integrate-community-build/6173/

File: report/Report.scala [hunk 2]
```diff
@@ -66,14 +66,23 @@ object SuccessReport {
...
+    "splain",
+    "scala-rewrites",
+    "scala-refactoring",
+  )
+
   val expectedToFail: Set[String] =
     System.getProperty("java.specification.version") match {
       case "1.8" =>
-        jdk8Failures
+        jdk8Failures ++ scala_2_13_4_Failures
```
Comment (SethTisue) [inline report/Report.scala]: You mean 2_12_13 :-)

Otherwise LGTM for squash + merge

**Pull request merged**